Clinical-Genomics/cgbeacon2
# Fix dockerfiles so containerized app worker is owner of workdir

### This PR adds | fixes:
- Fix #227 
 
**How to prepare for test**:
- Install the branch on cg-vm1

### How to test:
- Send a POST request with genes filter to the dockerized app and make sure that adding variants works


### Review:

This [version](https://semver.org/) is a:
